public/components/event_analytics/explorer/events_views/data_grid.tsx

```diff
@@ -114,8 +123,12 @@ export function DataGrid(props: DataGridProps) {
   const dataGridColumns = () => {
     const columns: EuiDataGridColumn[] = [];
     selectedColumns.map(({ name }) => {
-      if (name === 'timestamp') {
-        columns.push(DEFAULT_TIMESTAMP_COLUMN);
+      if (name === defaultTimestamp) {
+        columns.push({
+          ...DEFAULT_TIMESTAMP_COLUMN,
```

Comment:
I see that you are only overriding display and id, but there's also a schema in DEFAULT_TIMESTAMP_COLUMN as 'datetime', will this be a problem if the defaultTimestamp is not a schema of 'datetime'?

Reply:
the schema 'datetime' is a value specific to DataGrid, it's the only time related schema there is and assigning it to something that's not time related wouldn't cause anything to break.